content/community/devops/roadmapprocess/_index.nb.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Roadmap prosess
+description: Hvordan jobber vi med vår roadmap
+---
+Dette dokumentet beskriver hvordan vi jobber med features for Altinn 3 plattformen.
+
+## Artifakter
+
+Arbeidet med Altinn 3 krever en del artifakter.
+
+![Backlogs](roadmap.drawio.svg "Artifakter")
+[Fullscreen](roadmap.drawio.svg)
+
+### Feature request
+
+En feature request er en forespørsel om ønsket funksjonalitet i Altinn 3 plattformen. Den opprettes enten av eksterne
+som bruker plattformen, eller av andre som har identifsert et behov.
+
+Feature request opprettes i de forsjellige produktbackloggene.
+
+- [Altinn Studio](https://github.com/Altinn/altinn-studio/labels/kind%2Ffeature-request)
+- [Altinn Platform](https://github.com/Altinn/altinn-platform/labels/kind%2Ffeature-request)
+- [App Template](https://github.com/Altinn/app-template-dotnet/issues?page=2&q=is%3Aissue+is%3Aopen)
+
```

Comment:
Burde vi ha en egen entry for [app-frontend](https://github.com/Altinn/app-frontend-react) også?